source/slang/slang-ir-inst-defs.h

```diff
@@ -155,6 +150,11 @@ INST(Nop, nop, 0, 0)
             /* TextureTypeBase */
                 /* TextureType */
                 INST(TextureType, TextureType, 8, HOISTABLE)
+                INST(TextureShape1DType, TextureShape1DType, 0, HOISTABLE)
+                INST(TextureShape2DType, TextureShape1DType, 0, HOISTABLE)
+                INST(TextureShape3DType, TextureShape1DType, 0, HOISTABLE)
+                INST(TextureShapeCubeType, TextureShape1DType, 0, HOISTABLE)
+                INST(TextureShapeBufferType, TextureShapeBufferType, 0, HOISTABLE)
```

Comment:
Why did they become a part of TextureTypeBase?

Reply:
likely don't need the change now regardless